lib/tasks/role.rake

```diff
@@ -1,4 +1,13 @@
-desc 'Say hello!'
-task :hello_world do
-  puts "Hello"
-end
\ No newline at end of file
+ task create_role: :environment do
```

Comment:
Did you run this task locally?

Reply:
yes ma and it created the date in the db